File: W1/square2rev.c
#include <stdio.h>

int main() {
    int size;
    
    scanf("%d", &size);
    
    for ( int column = 0, total = size * size; column < size; column++ ) {
        for ( int row = 1, counter = total - size; row < size; row++ ) {
            counter += 1;
            printf("%d ", counter);
        }
        printf("%d\n", total);
        total -= size;
    }
    
    return 0;
}

File: PHP/square2rev.php
<?php
    $size = fgets(STDIN);

    for ( $column = 0, $total = $size * $size; $column < $size; $column++ ) {
        for ( $row = 1, $counter = $total - $size; $row < $size; $row++ ) {
            $counter += 1;
            echo $counter . ' ';
        }
        echo $total . PHP_EOL;
        $total -= $size;
    }
?>
